src/client/Microsoft.Identity.Client/ManagedIdentity/V2/FriendlyNameCodec.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Identity.Client.ManagedIdentity.V2
+{
+    /// <summary>
+    /// Encodes/decodes the persisted X.509 FriendlyName for MSAL mTLS certs.
+    /// Format: "MSAL|alias=cacheKey|ep=endpointBase"
+    /// Open the cert store and look at FriendlyName to see examples.
+    /// Wish we could paste a screenshot here... Maybe I can show it in code walkthroughs.
```

Comment:
?

Reply:
removed it - sometimes i just talk to myself I think